packages/userstorage/src/browser/user-storage-service-filesystem.ts

```diff
@@ -18,38 +18,34 @@ import { DisposableCollection, ILogger, Emitter, Event } from '@theia/core/lib/c
 import { UserStorageChangeEvent, UserStorageService } from './user-storage-service';
 import { injectable, inject } from 'inversify';
 import { FileSystemWatcher, FileChangeEvent } from '@theia/filesystem/lib/browser/filesystem-watcher';
+import { EnvVariablesServer } from '@theia/core/lib/common/env-variables';
 import { FileSystem } from '@theia/filesystem/lib/common';
 import URI from '@theia/core/lib/common/uri';
 import { UserStorageUri } from './user-storage-uri';
 
-export const THEIA_USER_STORAGE_FOLDER = '.theia';
-
 @injectable()
 export class UserStorageServiceFilesystemImpl implements UserStorageService {
 
     protected readonly toDispose = new DisposableCollection();
     protected readonly onUserStorageChangedEmitter = new Emitter<UserStorageChangeEvent>();
-    protected readonly userStorageFolder: Promise<URI | undefined>;
```

Comment:
this line should be reverted, `readonly` is correct, variable is assigned in the constructor

Reply:
ok